### ISSUE TITLE
*motorbabbling=false not working for UDPNAR

### ISSUE THREAD

**Issue by ntoxeg** [opennars/OpenNARS-for-Applications#164]
It seems that sending `*motorbabbling=false` to UDPNAR still causes it to babble with predefined operations, like `^say`.

**Comment by patham9** [comment]
I tried to replicate this, but failed.
` ./NAR UDPNAR 127.0.0.1 50000 100000 true`
then
`printf "*motorbabbling=false\0" | nc -w 1 -u 127.0.0.1 50000 &`
then repeatedly
`printf "G! :|:\0" | nc -w 1 -u 127.0.0.1 50000 &`
does not lead to any explorative actions.
I tried the same on Mac OS and Linux, in both cases sending *motorbabbling=false
worked. Sure your UDP packet didn't get lost for some reason?
